config/metrics.yml

```diff
@@ -15,67 +15,44 @@ metadata:
 - iterationsPerNamespace
 - jobIterations
 - jobType
-- maxWaitTimeout
 - metricsClosing
-- name
-- preLoadPeriod
-- qps
 - verifyObjects
 - waitForDeletion
 - waitWhenFinished
 - k8sVersion
 - masterNodesCount
 - masterNodesType
-- metricName
 - ocpMajorVersion
 - ocpVersion
 - passed
 - platform
-- publish
 - region
 - sdnType
 - totalNodes
-- uuid
 - workerArch
 - workerNodesCount
 - workerNodesType
+job_summary_filters:
+  platform.keyword: AWS
+  jobConfig.name.keyword: cluster-density-v2
 metrics:
-- alert
-# - avg-ro-apicalls-latency
-# - avg-mutating-apicalls-latency
-# - 99thEtcdDiskBackendCommit
-# - 99thEtcdDiskWalFsync
-# - 99thEtcdRoundTripTime
-# - cpu-kube-controller-manager
-# - max-memory-sum-kube-controller-manager
-# - cpu-etcd
-# - max-memory-sum-etcd
-- cpu-kube-apiserver
-- max-memory-sum-kube-apiserver
-- cpu-masters
-- max-memory-sum-masters
-- cpu-workers
-- max-memory-sum-workers
-- podLatencyQuantilesMeasurement
-benchmark: cluster-density-v2
-exclude_normalization:
-- LatencyMeasurement
-- jobSummary
-- nodeRoles
-- nodeStatus
-- alert
-- -start
-target_filters_by_data:
-- platform: AWS
-target_field_extract_filters:
```

Comment:
We need these when we want to extract fields based on a certain pattern. Mainly useful in case of fields that are reduced by labels.

Reply:
Aggregation will happen in the `reduce` section here we only do the field extraction part.